OxfordBodleian/Juel-Jensen/BDLaethc14.xml

```diff
@@ -52,147 +52,601 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                                stub between <locus target="#113"/> and <locus target="#114"/>
+                                            </item>
+                                            <item xml:id="q18" n="18">
+                                                <dim unit="leaf">9</dim>
+                                                <locus from="115r" to="123v"/>
+                                            </item>
+                                        </list>
+                                    </collation>
+                                    <condition key="deficient">
+                                        The textblock has been resewn together and parchment guards have been placed around all quires.
+                                        Leaves <locus from="51r" to="62v"/> have been stitched together secondarily.
+                                        The textblock is affected with humidity, especially in the outer margins, and water stains (e.g., on <locus target="#37 #104v"/>).
+                                        Wax stains are visible on <locus target="#31v #72v"/>. The parchment is stained and crumpled on <locus target="#74"/>.
+                                        Tears repaired on <locus target="#2 #35 #100 #106 #118"/>.
+                                    </condition>
+                                </supportDesc>
+                                
+                                <layoutDesc>
+                                    <layout columns="2" writtenLines="24">
+                                        <locus from="1ra" to="123vb"/>
+                                        <note corresp="#textarea #margin">Data on text area and margin dimensions taken from <locus target="#45r"/>.</note>
+                                        <!--      <dimensions unit="mm" xml:id="textarea1">
+                                        <height></height>
+                                        <width></width>
+                                       <!- - no text area provided: added to REQUEST FOR FURTHER ACTION list- ->
+                                    </dimensions>-->
+                                        <dimensions type="margin" unit="mm" xml:id="margin1">
+                                            <dim type="top">31</dim>
+                                            <dim type="bottom">56</dim>
+                                            <dim type="right">47</dim>
+                                            <dim type="left">15</dim>
+                                            <dim type="intercolumn">15</dim>
+                                        </dimensions>
+                                        <note> The characters per line are 14-17.</note>
+                                        <ab type="punctuation" subtype="Dividers">
+                                            Several caesura signs are attested in the manuscript:
```

Comment:
I have not heard of "caesura signs" is this attested? Otherwise, "marginal signs"?

Reply:
"Caesura sign" is used in Uhlig S., Introduction to Ethiopian Palaeography", 1990, on p. 59. Alternatively, Uhlig uses "caesura marker" on p. 38 while describing the christogram in the margin.
They translate the German equivalents "Makro- und Mikrozäsurzeichen" (p. 148 in Uhlig, Äthiop. Paläogr.) oder Zäsurelement (p. 277).

Reply:
I've never come across it in English, but as long as it's attested we are fine :)